core/app/models/spree/order/payments.rb

```diff
@@ -31,10 +31,6 @@ def capture_payments!
           process_payments_with(:purchase!)
         end
 
-        def pending_payments
```

Comment:
@gmacdougall I wonder if in this particular case removing it would be better?  From Spree 2.3 to 2.4 this method silently changed from [checkout+pending payments](https://github.com/spree/spree/blob/v2.3.10/core/app/models/spree/order.rb#L370-L372) to [just pending payments](https://github.com/spree/spree/blob/v2.4.1/core/app/models/spree/order/payments.rb#L34-L36) so aside from being extraneous the method is now also dangerous if anyone *is* using/modifying it externally.  It might be better in this case to remove the method entirely and hope that that causes a hard failure that people will notice?  What do you think?